### PR TITLE
switch to use new ci user references

### PR DESCRIPTION
**Pull request by Rozzii**
switch to use new metal3ci CI user references.

### CODE REVIEW

File: jenkins/jobs/integration_tests.pipeline [hunk 1]
```diff
@@ -116,8 +116,8 @@ pipeline {
               currentBuild.result = 'FAILURE'
             }
           }
-          withCredentials([usernamePassword(credentialsId: 'airshipci_city_cloud_openstack_credentials', usernameVariable: 'OS_USERNAME', passwordVariable: 'OS_PASSWORD')]) {
-            withCredentials([sshUserPrivateKey(credentialsId: 'airshipci_city_cloud_ssh_keypair', keyFileVariable: 'AIRSHIP_CI_USER_KEY')]){
+          withCredentials([usernamePassword(credentialsId: 'metal3ci_city_cloud_openstack_credentials', usernameVariable: 'OS_USERNAME', passwordVariable: 'OS_PASSWORD')]) {
+            withCredentials([sshUserPrivateKey(credentialsId: 'metal3ci_city_cloud_ssh_keypair', keyFileVariable: 'AIRSHIP_CI_USER_KEY')]){
```
Comment (fmuyassarov) [inline jenkins/jobs/integration_tests.pipeline:120]:
```suggestion
            withCredentials([sshUserPrivateKey(credentialsId: 'metal3ci_city_cloud_ssh_keypair', keyFileVariable: 'METAL3_CI_USER_KEY')]){
``` ?

File: jenkins/scripts/utils.sh [hunk 2]
```diff
@@ -6,16 +6,16 @@
 CI_EXT_NET="metal3-ci-ext-net"
 CI_EXT_SUBNET_CIDR="10.100.10.0/24"
 CI_FLOATING_IP_NET="ext-net"
-CI_METAL3_IMAGE="airship-ci-ubuntu-metal3-img"
-CI_METAL3_CENTOS_IMAGE="airship-ci-centos-metal3-img"
+CI_METAL3_IMAGE="metal3-ci-ubuntu-metal3-img"
+CI_METAL3_CENTOS_IMAGE="metal3-ci-centos-metal3-img"
```
Comment (fmuyassarov) [inline jenkins/scripts/utils.sh:10]: do we already have them?

File: jenkins/jobs/integration_tests.pipeline (hunk 1 above)
Comment (fmuyassarov) [reply jenkins/jobs/integration_tests.pipeline:120]: or just CI_USER

File: jenkins/jobs/bml_integration_tests.pipeline [hunk 3]
```diff
@@ -27,7 +27,7 @@ pipeline {
   }
   agent { label 'metal3-static-workers' }
   environment {
-    AIRSHIP_CI_USER="metal3ci"
+    METAL3_CI_USER="metal3ci"
```
Comment (fmuyassarov) [inline jenkins/jobs/bml_integration_tests.pipeline:30]: I am actually surprised that we have this variable here + in line 81. I think this is redundant, but let's not tackle it right now.

File: jenkins/jobs/integration_tests.pipeline (hunk 1 above)
Comment (Rozzii) [reply jenkins/jobs/integration_tests.pipeline]: done

File: jenkins/scripts/utils.sh (hunk 2 above)
Comment (Rozzii) [reply jenkins/scripts/utils.sh]: reverted

**Review by lentzi90 — APPROVED** [review]
LGTM

**Comment by fmuyassarov** [comment]
Currently we have a common issue with Centos integration tests and the failure on this PR is not related to the changes. In parallel to that, Ubuntu jobs are failing sometimes due the mismatch between old airshipci and new metal3ci users. As for that reason, we need to merge this PR manually even though it has failing Centos.
/hold cancel

**Pull request merged**